src/pages/Vote/testIds.ts

```diff
@@ -0,0 +1,10 @@
+const TEST_IDS = {
+  votingWallet: {
+    votingWeightValue: 'vote-voting-wallet-voting-weight-value',
+    totalLockedValue: 'vote-voting-wallet-total-locked-value',
+  },
+  governance: {
+    governanceProposal: (id: string) => `vote-governance-governance-proposal-${id}`,
+  },
+};
+export default TEST_IDS;
```

Comment:
We now have a file containing all the test IDs used in the app: `constants/tedIds`, these should be added to it

Reply:
This would be more correct considering the usage and naming is localized to the page per the pattern of keeping variables not used globally with the component

Other examples of this includes, hooks, constants and utility functions

Reply:
This is used globally, since multiple tests from different pages and component will potentially use this file